pkg/controller/clusterdeployment/clusterdeployment_controller_test.go

```diff
@@ -245,6 +245,9 @@ func testClusterDeployment() *hivev1alpha1.ClusterDeployment {
 				},
 			},
 		},
+		Status: hivev1alpha1.ClusterDeploymentStatus{
+			Installed: true,
```

Comment:
Are you using this to create an object through the API, or just reading the struct in code?